xml/cap_depl_azure.xml

```diff
@@ -450,13 +470,41 @@ Private IPs:\t\t\"$(az network nic list --resource-group $MCRGNAME -o json | jq
    </step>
   </procedure>
  </sect1>
- <sect1 xml:id="sec.cap.sample-config">
-  <title>Example &productname; Configuration File</title>
-
+<sect1 xml:id="sec.cap.tiller-service">
+    <title>Create &tiller; Service</title>
+    <para>
+        You must create a &tiller; service in order to give it sufficient
+        permissions to make changes on your cluster. Run these commands
+        to create your &tiller; service:
+    </para>
+    <screen>&prompt.user;kubectl -n kube-system create serviceaccount tiller
+ serviceaccount "tiller" created
+
+&prompt.user;kubectl create clusterrolebinding tiller \
+--clusterrole cluster-admin \
```

Comment:
While `cluster-admin` is a pre-defined role in CaaSP, it either needs to be created manually on AKS or another role needs to be used; for QA clusters our AKS deployment script creates this [here](https://github.com/SUSE/cf-ci/blob/f79a83410635cfb55401d1917cbfb2cc378d2bed/qa-tools/deploy-aks.sh#L127), using [this definition](https://github.com/SUSE/cf-ci/blob/f79a83410635cfb55401d1917cbfb2cc378d2bed/qa-tools/cluster-admin.yaml). Only the ClusterRole section of that needs to be deployed for the `kubectl create clusterrolebinding ...` instruction to work.